perun-core/src/main/java/cz/metacentrum/perun/core/implApi/MembersManagerImplApi.java

```diff
@@ -367,4 +367,15 @@
 	 * @throws SponsorshipDoesNotExistException if the given user is not sponsor of the given member
 	 */
 	void updateSponsorshipValidity(PerunSession sess, Member sponsoredMember, User sponsor, LocalDate newValidity) throws SponsorshipDoesNotExistException;
+
+	/**
+	 * Returns sponsorship, which have validityTo in range [from, to).
+	 * (from is inclusive, to is exclusive).
+	 *
+	 * @param sess session
+	 * @param from lower validityTo bound (inclusive)
+	 * @param to upper validityTo bound (exclusive)
+	 * @return list of sponsorships which have validityTo set in the given range
```

Comment:
You can add information about the lowest value for 'from' and highest value for 'to' (using MIN and MAX of date).

Reply:
I have updated the javadoc.